web/packages/teleport/src/Console/DocumentSsh/httpFileTransferHandlers.ts

```diff
@@ -74,6 +75,20 @@ export function getHttpFileTransferHandlers() {
       xhr.send();
       return eventEmitter;
     },
+    async getWebauthnResponse(
```

Comment:
`getWebauthnResponse` doesn't seem to be used anywhere.

Reply:
Vestigial, and removed